Libraries/LibVT/Terminal.cpp

```diff
@@ -165,7 +165,6 @@ void Terminal::SM(bool question_param, const ParamVector& params)
     alter_mode(false, question_param, params);
 }
 
-
```

Comment:
Please prefix all commit messages with *something*, e.g `"LibVT: "`

Reply:
Sure, forgot about this one. Rebased & fixed :)